src/components/general/NavBar.vue

```diff
@@ -6,7 +6,7 @@
 		<nav v-else>
 			<router-link to="/">HOME</router-link>
 			<router-link to="/speakers">SPEAKERS</router-link>
-			<!-- <router-link to="/schedule">SCHEDULE</router-link> -->
+			<router-link to="/schedule">SCHEDULE</router-link>
```

Comment:
Thanks for remembering to do this! But, also please uncomment line 21 for the mobile view :)